tests/unit/services/media-test.js

```diff
@@ -16,64 +15,64 @@ module('Unit | Service | media', function(hooks) {
   });
 
   test('matchers can be added dynamically', function(assert) {
-    var subject = this.owner.lookup('service:media');
-    run(() => {
-      subject.match('all', 'not all');
-    });
+    let subject = this.owner.lookup('service:media');
+    subject.match('all', 'not all');
 
     assert.equal(subject.get('matchers.all.matches'), false);
   });
 
   test('matchers have a corresponding isser', function(assert) {
-    var subject = this.owner.lookup('service:media');
-    run(() => {
-      subject.match('mobile', 'not all');
-    });
+    let subject = this.owner.lookup('service:media');
+
+    subject.match('mobile', 'not all');
 
     assert.equal(subject.get('isMobile'), false);
   });
 
   test('matches property returns matching matchers', function(assert) {
-    var subject = this.owner.lookup('service:media');
+    let subject = this.owner.lookup('service:media');
 
-    run(() => {
-      subject.match('mobile', 'all');
-      subject.match('all', 'all');
-      subject.match('none', 'not all');
-    });
+    subject.match('mobile', 'all');
+    subject.match('all', 'all');
+    subject.match('none', 'not all');
 
-    assert.deepEqual(subject.get('matches').toArray(), ['mobile', 'all']);
+    assert.deepEqual(subject.get('matches'), ['mobile', 'all']);
```

Comment:
Do you still need to use `subject.get('matches')` or could you use `subject.matches` now?

Reply:
Great idea!